testing/bdd/test_required_metadata.py

```diff
@@ -0,0 +1,12 @@
+import xml.etree.ElementTree as ET
+from pytest_bdd import scenarios, then
+
+scenarios('features/metadata/ebuttd_required_metadata.feature')
+
+
+@then('the EBUTTD document contains a documentMetadata element <element_name> with value <element_value>')
+def then_ebuttd_metadata_contains(test_context, element_name, element_value):
+    document = test_context['ebuttd_document']
+    tree = ET.fromstring(document.get_xml())
```

Comment:
This is an interesting approach - you're serialising the XML document as a string, then parsing it again using a different library to find the elements you want. I like it, because it removes the dependency on pyxb for verifying the tests. I'm pretty sure we don't do it anywhere else though. Seems like good practice for future similar tests.

Reply:
Yeah I didn't notice it anywhere else, but I felt that the test should really be verifying that the XML contains the right elements. Seemed to work nicely too.